README.md

```diff
@@ -285,40 +317,41 @@ Usage of ./sriovdp:
 
 For example, if the driver type is uio (i.e. igb_uio.ko) then there are specific device files to add in Device Spec. For vfio-pci, device files are different. And if it is Linux kernel network driver then there is no device file to be added.
 
 The idea here is, user creates a resource config for each resource pool as shown in [Config parameters](#config-parameters) by specifying the resource name, a list resource "selectors".
 
-The device plugin will initially discover all PCI network resources in the host and populate an initial "device list". Each "resource pool" then applies its selectors on this list and add devices that satisfies the selector's constraints. Each selector narrows down the list of devices for the resource pool. Currently, the selectors are applied in following order:
+The device plugin will initially discover all PCI network resources in the host and populate an initial "device list". If device type is Auxiliary network device (auxNetDevice), then for each discovered PCI device of type Netdevice plugin discovers auxiliry devices. Each "resource pool" then applies its selectors on this list and add devices that satisfies the selector's constraints. Each selector narrows down the list of devices for the resource pool. Currently, the selectors are applied in following order:
 
 1. "vendors"      - The vendor hex code of device
 2. "devices"      - The device hex code of device
 3. "drivers"      - The driver name the device is registered with
-4. "pciAddresses" - The pci address of the device in BDF notation
-4. "pfNames"      - The Physical function name
-5. "rootDevices"  - The Physical function PCI address
-6. "linkTypes"    - The link type of the net device associated with the PCI device.
+4. "pciAddresses" - The pci address of the device in BDF notation (if device type is accelerator or netDevice)
+5. "auxTypes"     - The type of auxiliary network device (if device type is auxNetDevice).
+6. "pfNames"      - The Physical function name
+7. "rootDevices"  - The Physical function PCI address
+8. "linkTypes"    - The link type of the net device associated with the PCI device
 
-The "pfNames" and "rootDevices" selectors can be used to specify a list and/or range of VFs for a pool in the below format:
+If device functions supports indexing (like Netdevice Virtual functions or Auxiliary netdevice Scalable functions), then "pfNames" and "rootDevices" selectors can be used to specify a list and/or range of functions for a pool in the below format:
```

Comment:
if we already define at the top Auxiliary netdevice Scalable functions as SF
then here you can just write:

`The "pfNames" and "rootDevices" selectors can be used to specify a list and/or range of VFs/SFs for a pool in the below format`

Reply:
Ok